test/e2e/onFCP-test.js

```diff
@@ -136,6 +122,10 @@ describe('onFCP()', async function () {
 
     await navigateTo('/test/fcp?hidden=1', {readyState: 'interactive'});
 
+    // Wait a bit to ensure an initial "frame" happens to avoid Safari
+    // flakiness with this test if switching visibility as soon as interactive.
+    await browser.pause(100);
+
```

Comment:
This is another one you might have issues with @philipwalton , but seems to help.

Reply:
Checked and even with `complete` Safari doesn't always finish loading web-vitals.js before changing the readyState. So the hardcoded pause seems to be the only thing that makes it more stable. Updated comment to reflect this.